api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +683,169 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
 	}
 	
 	/**
-	 * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnTrueWhenFolderContainsFileWithGivenName() {
+		File folder = createFolderInUserDirectory("folder");
+		String separator = FileSystems.getDefault().getSeparator();
+		File file = new File(folder.getPath()  + separator + "file");
+		try {
+			file.createNewFile();
+			assertTrue(OpenmrsUtil.folderContains(folder, "file"));
+			file.delete();
+
+		} 
+		catch (IOException e) {}
+		folder.delete();
+	}
+	
+	/**
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnFalseWhenFolderDoesNotContainFileWithGivenName() {
+		File folder = createFolderInUserDirectory("folder");
+		assertFalse(OpenmrsUtil.folderContains(folder,"file"));
+		folder.delete();
+	}
+	
+	/**
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnFalseWhenGivenStringIsNull() {
+		File folder = createFolderInUserDirectory("folder");
+		assertFalse(OpenmrsUtil.folderContains(folder, null));
+		folder.delete();
+	}
+	
+	/**
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnFalseWhenGivenFolderIsNull() {
+		assertFalse(OpenmrsUtil.folderContains(null,"file"));
+	}
+	
+	/**
+	 * @see OpenmrsUtil#conceptListHelper(String)
+	 */
+	@Test
+	public void conceptListHelper_shouldReturnEmptyListWhenGivenDescriptorIsNullOrEmpty() {
+		List<Concept> list = OpenmrsUtil.conceptListHelper("");
+		assertTrue(list.size() == 0);
```

Comment:
Doesn't `List` as a Collection have a defined method to check whether it's empty or not?

Reply:
Do we even need the variable! Isn't this just enough?
`assertTrue(OpenmrsUtil.conceptListHelper("").isEmpty());`

Reply:
Ok . I will do it like that .